src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -26,33 +27,50 @@ object Scope {
     def getChildrenList(ast: Ast): Seq[Ast] = Option(ast.getAttribute("body")) collect {
       case list: AstList => list.asScala.toVector
     } getOrElse Seq.empty collect {
-      case ast: Ast => ast
+      case node: Ast
+        if node.getName == AstNodeName.Call ||
+          node.getName == AstNodeName.Scatter =>
+        node
+    }
```

Comment:
I hate nitpicking whitespace, but shouldn't all the filtering be on a single line?